From 898981384da8beee4055d0d252e2c0b7a4f7da2c Mon Sep 17 00:00:00 2001
From: Tudor Ambarus <tudor.ambarus@microchip.com>
Date: Thu, 6 Oct 2022 12:39:11 +0300
Subject: [PATCH] configs: sama5d2: Update SDRAM offsets to cope with OP-TEE
 memory map

Update SDRAM offsets on sama5d2_xplained and sama5d27_som1_ek boards in
order to cope with OP-TEE memory map. OP-TEE needs the address space
between 0x20000000 and 0x22000000.

Signed-off-by: Tudor Ambarus <tudor.ambarus@microchip.com>
Co-developed-by: Sergiu Moga <sergiu.moga@microchip.com>
Signed-off-by: Sergiu Moga <sergiu.moga@microchip.com>
---
 include/configs/sama5d27_som1_ek.h | 2 +-
 include/configs/sama5d2_xplained.h | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/configs/sama5d27_som1_ek.h b/include/configs/sama5d27_som1_ek.h
index af23419dc2..71fc106083 100644
--- a/include/configs/sama5d27_som1_ek.h
+++ b/include/configs/sama5d27_som1_ek.h
@@ -18,7 +18,7 @@
 #define CONFIG_SYS_INIT_SP_ADDR		0x218000
 #else
 #define CONFIG_SYS_INIT_SP_ADDR \
-	(0x20000000 + 16 * 1024 + CONFIG_SYS_MALLOC_F_LEN - \
+	(0x22000000 + 16 * 1024 + CONFIG_SYS_MALLOC_F_LEN - \
 	 GENERATED_GBL_DATA_SIZE)
 #endif
 
diff --git a/include/configs/sama5d2_xplained.h b/include/configs/sama5d2_xplained.h
index 8d65f4ffed..3b4cfa3e09 100644
--- a/include/configs/sama5d2_xplained.h
+++ b/include/configs/sama5d2_xplained.h
@@ -15,7 +15,7 @@
 #define CONFIG_SYS_INIT_SP_ADDR		0x218000
 #else
 #define CONFIG_SYS_INIT_SP_ADDR \
-	(0x20000000 + 16 * 1024 + CONFIG_SYS_MALLOC_F_LEN - \
+	(0x22000000 + 16 * 1024 + CONFIG_SYS_MALLOC_F_LEN - \
 	 GENERATED_GBL_DATA_SIZE)
 #endif
 
-- 
2.25.1

